apps/desktop/src/renderer/components/library/series/chapter-table/ChapterTable.tsx

```diff
@@ -311,75 +333,197 @@ export function ChapterTable(props: ChapterTableProps) {
     downloaderClient.start();
   };
 
+  const downloadAll = () => {
+    downloaderClient.add(
+      getAllChapters().map((chapter) => ({
+        chapter,
+        series: props.series,
+        downloadsDir: customDownloadsDir || defaultDownloadsDir,
+      })),
+    );
+    downloaderClient.start();
+  };
+
   return (
     <div className="space-y-2 pb-4">
       <div className="flex items-center justify-between">
-        {table.getIsSomeRowsSelected() || table.getIsAllRowsSelected() ? (
-          <div className="flex space-x-2 items-end">
-            <Button className="ml-auto" onClick={() => setSelectedRead(true)}>
-              <Eye className="w-4 h-4" />
-              Mark selected read
-            </Button>
-            <Button className="ml-auto" onClick={() => setSelectedRead(false)}>
-              <EyeOff className="w-4 h-4" />
-              Mark selected unread
-            </Button>
-            {/* TODO add confirmation prompt */}
-            <Button className="ml-auto" onClick={() => downloadSelected()}>
-              <Download className="w-4 h-4" />
-              Download selected
-            </Button>
-          </div>
-        ) : (
-          <>
-            <div className="flex space-x-2">
-              <ChapterTableLanguageFilter />
-              <ChapterTableGroupFilter
-                uniqueGroupNames={Array.from(
-                  new Set(chapterList.map((chapter) => chapter.groupName)),
-                )}
-              />
-            </div>
-            <div className="flex space-x-2">
-              <DropdownMenu>
-                <DropdownMenuTrigger asChild>
-                  <Button variant="outline" className="ml-auto">
-                    <Settings2 className="w-4 h-4" />
-                    View
-                  </Button>
-                </DropdownMenuTrigger>
-                <DropdownMenuContent align="end">
-                  <DropdownMenuLabel>Columns</DropdownMenuLabel>
-                  <DropdownMenuSeparator />
-                  {table
-                    .getAllColumns()
-                    .filter((column) => column.getCanHide())
-                    .map((column) => {
-                      return (
-                        <DropdownMenuCheckboxItem
-                          key={column.id}
-                          className="capitalize"
-                          checked={column.getIsVisible()}
-                          onCheckedChange={(value) => column.toggleVisibility(!!value)}
-                          onSelect={(event) => event.preventDefault()}
+        <div className="flex space-x-2">
+          <DropdownMenu>
+            <DropdownMenuTrigger asChild>
+              <Button variant="outline">
+                <Filter className="w-4 h-4" />
+                Filters
+              </Button>
```

Comment:
doesn't need to be part of this PR, but I think it would be good to have a badge with `X selected` on the button when either language/group filters are applied.